modules/about-rhde.adoc

```diff
@@ -34,29 +34,29 @@ The latest release notes for each product that is part of {product-title} are av
 [id="device-edge-compatibility_{context}"]
 == {product-title} product release compatibility matrix
 
-{microshift-first} and {op-system} work together as a single solution for device-edge computing. To successfully pair your products, use the verified releases together for each as listed in the following table:
+{op-system-first} and {microshift} work together as a single solution for device-edge computing. You can update each component separately, but the versions must be compatible. An update of {microshift} from 4.14 to 4.16 requires a {op-system} update, for example. Supported configurations of {product-title} use verified releases for each together as listed in the following table:
```

Comment:
```suggestion
{op-system-first} and {microshift} work together as a single solution for device-edge computing. You can update each component separately, but the product versions must be compatible. For example, an update of {microshift} from 4.14 to 4.16 requires a {op-system} update. Use the following table to verify the version compatibility of {op-system-first} and {microshift} for {product-title} use:
```

Reply:
A few suggestions here:

- I think it helps to add "product" before versions
- I recommend switching "for example" to the beginning instead of the end, that way the customers know what the sentence is about from the beginning. 
- I recommended a re-working of the table introduction. Mostly, I was having trouple with "Supported configurations...use verified release for each together". It just seemed really confusing to me, and I _think_ it should mention compatibility instead of support just because you use compatibility to describe the version updates previously.

Reply:
@skrthomas "Support" is a keyword that is critical to the nature of RHDE and should not be changed. This is more than compatibility, these are components of a single, supported SKU that is Red Hat Device Edge.

Reply:
@ShaunaDiaz Understood. I guess maybe the "each together" phrasing just came off as a bit jumbled to my eyes. 

Maybe instead
> The following table lists the supported configurations of Red Hat Device Edge, using verified releases for each _product_ together.

Reply:
1st and 2nd bullet point suggestions are done. 
3rd line is kept as is as per my discussion with Shauna. Also, I have not made edits to this line and I have ported this line. So it is not in scope of this PR.